src/harness/reference_models/iap/test_data/cbsd_3.json

```diff
@@ -0,0 +1,44 @@
+{
+        "_comment":"PPA - CBSD and Category B - ESC CBSD",
+	"id": "cbsd_3",
+	"registrationRequest": {
+					"fccId": "test_fcc_id_d",
+		            "cbsdCategory": "B",
```

Comment:
Should not be Category B and the indoorDeployment = true

Reply:
Updated